cypress/e2e/filter/filterChips.cy.ts

```diff
@@ -292,6 +295,55 @@ describe("Case list", () => {
           .should("exist")
       })
     })
+
+    describe('Applied filter chips to "Filter applied" section', () => {
+      it("Should display the Trigger filter chip when selected", () => {
+        cy.get("#filter-button").click()
+        cy.get(".govuk-checkboxes__item").contains("Triggers").click()
+
+        // Check if the correct heading and filter label are applied
+        cy.get(".govuk-heading-s").contains("Reason").should("exist")
+        cy.get(".moj-filter__tag").contains("Triggers").should("exist")
+        cy.get(".moj-filter__tag").contains("Exceptions").should("not.exist")
+
+        // Check if the filter chip is applied to the "Filters applied" section at the top of the case list
+        cy.contains("Apply filters").click()
+        cy.get(".moj-filter-tags").contains("Triggers")
```

Comment:
I think this wants a `.should("exist")` assertion at the end of the line

Reply:
Do we need it? I was removed for simplicity, I think that if `"Triggers"` was not there, this would fail. Shall we put this back in?

Reply:
We've added it for visibility.